### ISSUE TITLE
Adhoc Toolkits

### ISSUE THREAD

**Issue by loklaan** [reactbris/meetup#44]
Investing in the developer experience of your _app bootstrapping & build process_ pays when frequently creating similar products.

The lovely @colingourlay, as part of his work at @abcnews, has discovered the benefits of creating your [own company toolkit](https://github.com/abcnews/aunty) along with the concerns surfaced along the way.

**Comment by julianburr** [comment]
As mentioned I'd be keen to do a talk on toolkits at the May meetup, essentially a general introduction what toolkits are, why they are super useful, why react-scripts and cra are so great and why you should still write your own 😅

Ideally after that we could have a panel of people with experience / opinions to answer everyone's questions.

Thoughts?

**Comment by loklaan** [comment]
Yeah I'm down. Did you imagine having a panel in the same meetup (may?).

**Comment by julianburr** [comment]
I think if possible it would make sense, so people who have questions or wanna get a bit more details after the talk have a bunch of people to aim those questions at, in a separate meetup would mean it would be kind of out of context I guess

Just a wish/suggestion tho :smile:

**Comment by loklaan** [comment]
No, you're right.

When thinking about the kind of diversity in people and tech for a panel like this, I imagine it being broader than React. 🤔 It might be more suitable, as a Panel, to be a part of @brisjs.

But as a talk, reactbris certainly needs it imo. Idk, maybe I'm getting in my own way here haha.

@iamtommcc third amigo thoughts?
